game/static/game/css/landing.css

```
@@ -1675,6 +1675,118 @@ body.hide-native-cursor * {
+
+.stat-icon {
+    font-size: 1.75rem;
+    margin-bottom: 0.75rem;
+    filter: drop-shadow(0 2px 8px rgba(0, 0, 0, 0.2));
+}
+
+.stat-number-wrapper {
+    display: flex;
+    align-items: baseline;
+    justify-content: center;
+    font-family: 'Exo 2', 'Rajdhani', sans-serif;
```

Stylelint (17.13.0)

[error] Expected no quotes around "Rajdhani" (font-family-name-quotes)

(font-family-name-quotes)